web/server/vue-cli/src/components/Report/SelectReviewStatus.vue

```diff
@@ -57,6 +57,53 @@
 
       <v-card-text class="pa-0">
         <v-container>
+          <v-alert
```

Comment:
I think it will be good to inform the user when changing the review status on the GUI that if the user sets the review status to False positive or Intentional the report fixed date will be set to the current time. We can app this information to this pop-up window:
![image](https://user-images.githubusercontent.com/6695818/162931621-a3308161-b440-4bbd-b054-33f9296f24b2.png)

Maybe below the message input area.